server/src/session.ts

```diff
@@ -882,6 +892,14 @@ export class Session {
     if (!definitions) {
       return null;
     }
+
+    const clientSupportsLocationLinks =
+        this.clientCapabilities.textDocument?.definition?.linkSupport ?? false;
```

Comment:
```suggestion
        this.clientCapabilities.textDocument?.typeDefinition?.linkSupport ?? false;
```

nit: I think technically this should be `typeDefinition` though I doubt these would ever truly be different in the client capabilities. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_typeDefinition